src/SignalRMock.ts

```diff
@@ -1,8 +1,7 @@
-import { subscribers } from './subscribers';
+import { subscribers, removeSubscriberByAction, removeSubscriberByCallback } from './subscribers';
 import { invokes } from './invokes';
 
 export class SignalRMock {
-
   on(action: string, callback: (...args: any) => void) {
```

Comment:
While your at it, could you add the return types for the methods? Like `on(...): Promise<void> {}` and `off(...): void {}` etc.

Reply:
Done.